fe/fe-core/src/test/java/org/apache/doris/nereids/rules/exploration/join/JoinCommuteTest.java

```diff
@@ -47,7 +48,8 @@ public void testInnerJoinCommute() {
                 new SlotReference("id", new BigIntType(), true, ImmutableList.of("table1")),
                 new SlotReference("id", new BigIntType(), true, ImmutableList.of("table2")));
         LogicalJoin<LogicalOlapScan, LogicalOlapScan> join = new LogicalJoin<>(
-                JoinType.INNER_JOIN, Optional.of(onCondition), scan1, scan2);
+                JoinType.INNER_JOIN, Lists.newArrayList(onCondition),
```

Comment:
we need to add case that join has other conjuncts

Reply:
done